lib/modules/touch.js

```diff
@@ -0,0 +1,50 @@
+import { contentLoaded } from '../utils';
+
+export default (shower) => {
+    contentLoaded(() => {
+        let action = false;
+        let clickable = false;
+
+        document.addEventListener('touchstart', (ev) => {
+            if (ev.touches.length > 1) {
+                action = false;
+                return true;
+            }
+            const touch = ev.touches[0];
+            const x = touch.clientX;
+            const { target } = touch;
+            clickable =
+                target.getAttribute('onclick') != null || target.getAttribute('href') != null;
+            if (!clickable) {
+                if (target.tagName.toLowerCase() === 'html') {
+                    if (ev.cancelable) ev.preventDefault();
```

Comment:
Isn't `touchstart` always cancelable?

Reply:
Yes. On Safari if you touch only one finger, then `touchstart` can't cancelable